api/src/main/resources/liquibase.xml

```diff
@@ -987,4 +987,17 @@
         <dropTable tableName="fhir_duration_unit_map" />
     </changeSet>
 
+    <changeSet id="update_fhir_diagnostic_report_table_20220511" author="moses_mutesa" dbms="mysql,mariadb,postgresql">
+        <preConditions onFail="MARK_RAN" onError="WARN">
+            <tableExists tableName="fhir_diagnostic_report"/>
+            <columnExists tableName="fhir_diagnostic_report" columnName="name"/>
+            <not>
+                <columnExists tableName="fhir_diagnostic_report" columnName="issued"/>
+            </not>
+        </preConditions>
+        <addColumn tableName="fhir_diagnostic_report">
+            <column name="issued" type="datetime"/>
+        </addColumn>
+        <dropColumn tableName="fhir_diagnostic_report" columnName="name"/>
```

Comment:
was the name column just erroneous?

Reply:
yeah , the name field is not inherited from the [BaseOpenmrsData object](https://github.com/openmrs/openmrs-module-fhir2/blob/master/api/src/main/java/org/openmrs/module/fhir2/model/FhirDiagnosticReport.java#L44) , its only inherited from **BaseOpenmrsMetaData** , so here its not populated any where and just results to errors